src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -1588,11 +1588,10 @@ public async Task<JObject> InvokeMethod(ArraySegment<byte> valueTypeBuffer, int
             return await CreateJObjectForVariableValue(retDebuggerCmdReader, varName, false, -1, false, token);
         }
 
-        public async Task<JObject> InvokeMethodInObject(int objectId, int methodId, string varName, CancellationToken token)
+        public async Task<JObject> InvokeMethodInObject(int objectId, int methodId, ElementType elementType, string varName, CancellationToken token)
```

Comment:
What we need to know here really is whether the objectId is for a valuetype or not. Use a `bool isValueType` instead.